Day3/application.py

```diff
@@ -9,15 +9,23 @@
 Session = sessionmaker(bind = engine)
 session = Session()
  
+@app.route("/admin")
+def admin():
+   usrs =   session.query(User).all()
+   # for i in usrs:
+      # print(i)
```

Comment:
remove commented code

Reply:
removed the code sir, will commit after all the changes.